r/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM arrow:cpp
+
+# get the stuff
+ADD r /arrow/r
+
+# Configure
+ENV CC=gcc \
+    CXX=g++
+
+# r-base includes tzdata. Get around interactive stop in that package
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Build R
+# [1] https://www.digitalocean.com/community/tutorials/how-to-install-r-on-ubuntu-18-04
+# [2] https://linuxize.com/post/how-to-install-r-on-ubuntu-18-04/#installing-r-packages-from-cran
+RUN apt install -y \
```

Comment:
I'm not really familiar with R packaging, but it'd be great if We could install `R`  and the dependencies via conda.
Is it possible?

Reply:
I think best to use CRAN

Reply:
Might be relevant to involve the rocker organization: https://github.com/rocker-org/rocker

Reply:
We have three: CRAN, conda, rocker options. A drawback of Rocker is it requires the duplication of the code from [cpp Dockerfile](https://github.com/apache/arrow/blob/master/cpp/Dockerfile).

Thanks to the per-language dockerfiles it's really up to YouR preference which one to use :)
>  I like to handle dependencies in a unified way, so I'd go with conda.

Reply:
Sounds good to me!